examples/step-92/step-92.cc

```diff
@@ -0,0 +1,1091 @@
+/* ---------------------------------------------------------------------
+ *
+ * This example can be compiled for usage with parallel::distributed or
+ * parallel::fullydistributed triangulations
+ *
+ * In order to force usage of parallel::fullydistributed triangulations
+ * define
+ * #define USE_FULLY_DISTRIBUTED_TRIA
+ *
+ * otherwise (if not defined), parallel::distributed triangulation
+ * will be used
+ * ---------------------------------------------------------------------
+ */
+#define USE_FULLY_DISTRIBUTED_TRIA
+
+/* ---------------------------------------------------------------------
+ *
+ * 2024-11, Stephan Voss, Neunkirchen am Brand, Germany, stvoss@gmx.de
+ *
+ * This solver is derived from several deal.II examples and tutorials.
+ * Thank You to all deal.II contributors.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/* ---------------------------------------------------------------------
+ *
+ * deal.II: Copyright (C) 2021 - 2023 by the deal.II authors
+ *
+ * The deal.II library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE.md at
+ * the top level directory of deal.II.
+ *
+ * ---------------------------------------------------------------------
+ */
```

Comment:
We'll have to restrict this to the standard headers for tutorials only, I think. But this can be done at the end of the review.